src/Components/CreateImageWizard/steps/Oscap/components/ProfileSelector.tsx

```diff
@@ -329,7 +329,7 @@ const ProfileSelector = () => {
     >
       <TextInputGroup isPlain>
         <TextInputGroupMain
-          value={profileID ? profileID : inputValue}
+          value={profileID ? profileID : inputValue || 'None'}
```

Comment:
If you try to remove the "None" value to filter the results it will just pop "None" back. I wonder if we could just replace the placeholder text by "None" in this case? As it's a default option.